cerebrium/data-sharing-storage/persistent-storage.mdx

```diff
@@ -2,52 +2,96 @@
+```bash
 
-# Load the model
-model = torch.jit.load(file_path)
-... # Continue with your initialization
-```
+Usage: cerebrium files cp [OPTIONS] LOCAL_PATH REMOTE_PATH (Optional)
 
-Now, in subsequent deployments, the model will load from the cache rather than download it again.
+  Copy contents to persistent storage.
 
-## Increasing your Persistent Storage Size
+Options:
+  -h, --help          Show this message and exit.
 
-<Note>Once increased, your persistent storage size cannot be decreased.</Note>
+Examples:
+  # Copy a single file
+  cerebrium files cp src_file_name.txt # copies to /src_file_name.txt
 
-By default, your account is given 50GB of persistent storage to start with. However, if you find you need more (for example, you get an error saying `disk quote exceeded`) then you can increase your allocation using the following steps:
+  cerebrium files cp src_file_name.txt dest_file_name.txt # copies to /dest_file_name.txt
 
-1. Check your current persistent storage allocation by running:
+  # Copy a directory
+  cerebrium files cp dir_name # copies to the root directory
+  cerebrium files cp dir_name sub_folder/ # copies to sub_folder/
+```
+
+### Listing files
+
+To list the files in your persistent storage, you can use the cerebrium files ls [remote_path] command. This command lists all files and directories within the specified remote_path. If no remote_path is provided, it lists the contents of the root directory of the persistent storage.
 
 ```bash
-cerebrium storage get-capacity
+Usage: cerebrium files ls [OPTIONS] REMOTE_PATH (Optional)
+
+  List contents of persistent storage.
+
+Options:
+  -h, --help          Show this message and exit.
+
+Examples:
+  # List all files in the root directory
+  cerebrium files ls
+
+  # List all files in a specific folder
+  cerebrium files ls sub_folder/
 ```
 
-This will return your current persistent storage allocation in GB.
+### Deleting files
+
+To delete files or directories from your persistent storage, use the cerebrium files rm remote_path command. This command removes the specified file or directory from the persistent storage. Be careful, as this operation is irreversible.
+
+```bash
+Usage: cerebrium files rm [OPTIONS] REMOTE_PATH
+
+  Remove a file or directory from persistent storage.
+
+Options:
+  -h, --help          Show this message and exit.
 
-2. To increase your persistent storage allocation run:
+Examples:
+  # Remove a specific file
+  cerebrium files rm /file_name.txt
+
+  # Remove a directory and all its contents
+  cerebrium files rm /sub_folder/
+```
+
+### Real world example
 
 ```bash
-cerebrium storage increase-capacity <number of GB to increase by>
+wget https://dl.fbaipublicfiles.com/segment_anything/sam_vit_h_4b8939.pth
+cerebrium cp sam_vit_h_4b8939.pth segment-anything/sam_vit_h_4b8939.pth
+```
+
+As a simple example, suppose you have an external SAM model that you want to use in your custom deployment. You can download it to the cache
```

Comment:
"download to cache"?

Reply:
+, should be more like "download to a cache directoy on your persistent volume